core/src/test/java/jackszm/androiddevtweets/tweets/TweetsServiceShould.java

```diff
@@ -0,0 +1,42 @@
+package jackszm.androiddevtweets.tweets;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import jackszm.androiddevtweets.Tweet;
+import rx.observers.TestSubscriber;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TweetsServiceShould {
+
+    private TweetsService tweetsService;
+
+    @Before
+    public void setUp() throws Exception {
+        tweetsService = new TweetsService();
+    }
+
+    @Test
+    public void loadTweets() {
+        TestSubscriber<List<Tweet>> testSubscriber = new TestSubscriber<>();
+
+        tweetsService.loadTweets().subscribe(testSubscriber);
+
+        assertThat(testSubscriber.getOnNextEvents().get(0)).isEqualTo(expectedTweets());
+    }
+
+    private List<Tweet> expectedTweets() {
```

Comment:
it depends how it will be implemented. Not decided yet. Probably will be calling some other collaborator or doing api call by itself. 

Anyways we would need to mock the response or something in those lines.